<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Movement Pawn</string>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUP">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>NORTH_WEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNE">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>NORTH_EST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSW">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SUD_WEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSE">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SUD_EST</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btnVerti">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btnHori">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlace">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Place Wall</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Placement Wall</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Current Player Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>53</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="btnCol">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>107</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="btnRow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Row</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlayerInfo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>The current Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of wall :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlayerWalls">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>btnUP</zorder>
   <zorder>btnLeft</zorder>
   <zorder>btnRight</zorder>
   <zorder>btnDown</zorder>
   <zorder>btnNW</zorder>
   <zorder>btnNE</zorder>
   <zorder>btnSW</zorder>
   <zorder>btnSE</zorder>
   <zorder>btnVerti</zorder>
   <zorder>btnHori</zorder>
   <zorder>btnPlace</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>btnCol</zorder>
   <zorder>btnRow</zorder>
   <zorder>label_3</zorder>
   <zorder>lblPlayerInfo</zorder>
   <zorder>label_6</zorder>
   <zorder>lblPlayerWalls</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Game">
    <property name="title">
     <string>Menu Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionQuit_Game"/>
   </widget>
   <addaction name="menuMenu_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionQuit_Game">
   <property name="text">
    <string>Quit Game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
